1.train_models/classify_genotypes/nbconverted/classify_genotypes.py

```diff
@@ -240,44 +224,11 @@ def create_splits(_wells, _plate):
 
 
 # Columns common to all plates
-plate_cols = list(
-    set(plate5df.columns)
-    & set(plate3df.columns)
-    & set(plate3pdf.columns)
-    & set(plate4df.columns)
-)
-
-restdf = pd.concat(
-    [
-        rest5df[plate_cols],
-        rest3df[plate_cols],
-        rest3pdf[plate_cols],
-        rest4df[plate_cols],
-    ],
-    ignore_index=True,
-).reset_index(drop=True)
-
-# Add Metadata_datasplit column to restdf
-restdf = restdf.assign(datasplit='rest')
-
-testdf = pd.concat(
-    [
-        test5df[plate_cols],
-        test3df[plate_cols],
-        test3pdf[plate_cols],
-        test4df[plate_cols],
-    ],
-    ignore_index=True,
-).reset_index(drop=True)
-
-# Add Metadata_datasplit column to testdf
-testdf = testdf.assign(datasplit='test')
-
-# Concatenate restdf and testdf vertically
-model_df = pd.concat([restdf, testdf], ignore_index=True)
-
-# Save model data to apply the model for evaluation
-model_df.to_parquet("./model_data.parquet")
+plate_cols = list(set(plate5df.columns) & set(plate3df.columns) & set(plate3pdf.columns) & set(plate4df.columns))
+
+restdf = pd.concat([rest5df[plate_cols], rest3df[plate_cols], rest3pdf[plate_cols], rest4df[plate_cols]], ignore_index=True).reset_index(drop=True)
+
+testdf = pd.concat([test5df[plate_cols], test3df[plate_cols], test3pdf[plate_cols], test4df[plate_cols]], ignore_index=True).reset_index(drop=True)
```

Comment:
Why do this step?? Have we considered performing feature selection on the combined plate dataset??

Reply:
This is an extra precaution to make sure the plate have the same columns after feature selection for training and evaluation. One way to see this is through the lens of having a very large feature space we are trying to condense. In this sense, if feature doesn't give much value as determined by our feature selection, then we wouldn't be at much of a loss for ignoring that feature entirely. Similarly, processing these plates simultaneously is more computationally restrictive (e.g. memory usage)